src/GraphQL/Schemas/Primary/Substrate/Queries/GetCollectionsQuery.php

```diff
@@ -68,10 +71,35 @@ public function resolve(
         ResolveInfo $resolveInfo,
         Closure $getSelectFields,
     ): mixed {
-        return Collection::loadSelectFields($resolveInfo, $this->name)
+        $result = Collection::loadSelectFields($resolveInfo, $this->name)
             ->addSelect(DB::raw('cast(collection_chain_id as unsigned integer) as collection_id'))
             ->when(!empty($args['collectionIds']), fn (Builder $query) => $query->whereIn('collection_chain_id', $args['collectionIds']))
             ->cursorPaginateWithTotalDesc('collection_id', $args['first']);
+
+        $items = $result['items']->items();
+        if (count(Arr::get($items, '0.tokens', []))) {
+            $totalCounts = Token::whereIn('collection_id', Arr::pluck($items, 'id'))
+                ->select('collection_id', DB::raw('count(*) as total'))
+                ->groupBy('collection_id')
+                ->pluck('total', 'collection_id');
+
+            foreach ($items as $item) {
+                $item->tokens_count = $totalCounts->get($item->id, 0);
+            }
+        }
+
+        if (count(Arr::get($items, '0.accounts', []))) {
+            $totalCounts = CollectionAccount::whereIn('collection_id', Arr::pluck($items, 'id'))
+                ->select('collection_id', DB::raw('count(*) as total'))
+                ->groupBy('collection_id')
+                ->pluck('total', 'collection_id');
+
+            foreach ($items as $item) {
+                $item->accounts_count = $totalCounts->get($item->id, 0);
+            }
+        }
+
+        return $result;
```

Comment:
```suggestion
        }
        
        $result['items']->setCollection(collect($items));

        return $result;
```
Should the `$items` be assigned back to the `$result` before returning it?  Or did I miss something 😅

Reply:
There's no need, object is passed by reference so altering the values will immediately reflect.